util.py

```diff
@@ -27,11 +27,49 @@
 
 @lru_cache(maxsize=32)
 def read_api_key(key_name: str) -> str:
-    """ Read the given key from a text file in resources directory. Cached. """
+    """Read the given key from a text file in resources directory. Cached."""
     path = os.path.join(os.path.dirname(__file__), "resources", key_name + ".txt")
     try:
         with open(path) as f:
             return f.read().strip()
     except FileNotFoundError:
         pass
     return ""
+
+
+def icelandic_asciify(text: str) -> str:
+    """Convert Icelandic characters to their ASCII equivalent
+    and then remove all non-ASCII characters."""
+
+    ICECHARS_TO_ASCII = {
+        "ð": "d",
+        "Ð": "D",
+        "á": "a",
+        "Á": "A",
+        "ú": "u",
+        "Ú": "U",
+        "í": "i",
+        "Í": "I",
+        "é": "e",
+        "É": "E",
+        "þ": "th",
+        "Þ": "TH",
+        "ó": "o",
+        "Ó": "O",
+        "ý": "y",
+        "Ý": "Y",
+        "ö": "o",
+        "Ö": "O",
+        "æ": "ae",
+        "Æ": "AE",
+    }
+
+    # Substitute all Icelandic chars for their ASCII equivalents
+    t = text
+    for k, v in ICECHARS_TO_ASCII.items():
```

Comment:
This can be done more efficiently with a regex (`re.sub()`), but perhaps it isn't worth the trouble.

Reply:
I was thinking the same thing while reading over this PR, and decided it wasn't worth the trouble.